client/src/components/PageList/PageList.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+const PageList = () => {
+
+  const onClick = async () => {
+
+    const token = (localStorage.getItem('userInfo')) ? JSON.parse(localStorage.getItem('userInfo')).token : null
+
+    const fetched = await fetch('/test',
```

Comment:
Naming is very confusing here. Maybe `response` instead of `fetched` and `value` or `result` instead of `post`?

Reply:
fixed